### ISSUE TITLE
In cinema view, can we hide the playbar and the three buttons?

### ISSUE THREAD

**Issue by fastnlight0** [surfbryce/beautiful-lyrics#343]
In the cinema view, there are three buttons:
![Photo of the three buttons](https://github.com/surfbryce/beautiful-lyrics/assets/77035115/5b28d6fb-1173-4ecb-b95e-65fc021c0015)
The far left one is the eyeball that hides the items to the left and right of the playbar.
Here's a screenshot of the hidden playbar details:
![Photo of the playbar with hidden playbar details on](https://github.com/surfbryce/beautiful-lyrics/assets/77035115/d134ec8e-9e3f-49dd-98c0-4adfe6a9af67)
As you can see, the playbar itself is still shown. Can we be able to hide that too?

In addition, when the cursor is above the Spotify window, the three buttons mentioned above are still visible, just with low opacity. Could those react similar to the playbar details (completely hidden until you hover over them)? That way, I can get a clean cinema view with nothing but lyrics. No buttons, no progress bar, just some beautiful lyrics.

**Comment by surfbryce** [comment]
@fastnlight0 For the play-bar request, I've thought about it in the past, I may make it so the visibility toggle can have levels to how much it hides in Cinema mode.

I don't really understand your last request though, what is your situation where you can't have the mouse off Spotify when it is in Cinema mode? Are you using Cinema mode on a single monitor and aren't able to move the mouse off of it because you have Spotify fullscreen or fully-scaled? Knowing this will help a lot, I don't think I'll implement something like it though because I don't really have a good concept for how I would even design the user-interface for hiding/showing the page-view controls without permanently losing them (and in a manner that is accessible and intuitive for everybody)

**Comment by fastnlight0** [comment]
For my final request, I'm saying if I have a single monitor in full screen, I could still get a clean look. Maybe do something where if the pointer is idle, it will hide the controls? Moving the cursor reveals the controls again. You could mix this with the play bar, so being idle will hide the play bar and controls. Moving the cursor reveals both.

**Comment by surfbryce** [comment]
@fastnlight0 Good idea, I might try that for the page-controls. Playbar will probably still use the same visibility toggle and just introduce levels to it.
